core/oWebSocketServer.php

```diff
@@ -455,11 +454,7 @@ private function fwrite_stream($socket, $string)
 		for ($written = 0; $written < strlen($string); $written += $fwrite) {
 
 			//		1.	attempt to write, handle error
-			try {
-				$fwrite = @fwrite($socket, substr($string, $written), 10240);
-			} catch (Exception $err) {
-				return FALSE;
```

Comment:
I don't believe this can throw an exception. It returns `false` on error, and I think there are other functions used to look up the errors that occurred.